src/Facebook/FacebookClient.php

```diff
@@ -76,9 +76,9 @@ class FacebookClient
     protected $enableBetaMode = false;
 
     /**
-     * @var FacebookHttpClientInterface HTTP client handler.
+     * @var HttpClient HTTP client handler.
      */
-    protected $httpClientHandler;
+    protected $httpClient;
```

Comment:
BC break to be documented - consider switching to a `private` property for future development